### PR TITLE
2021 Drink Revamp

### PR DESCRIPTION
**Pull request by GhostActual**
Polishes, adds, and changes some sprites for both cans and bottles alike that were a bit outdated. Also re-adds the Radical Renard vending machine with a more appeasing sprite that is also partially animated. Some organizational changes were also added to the code for better reference and a `can_placeholder` was added to the `drinks.dmi` file for any future canned drink additions.

Below are some examples of the sprites that were changed/added.

Placeholder Can:
placeholder

Various Bottle Changes:
asgasg

Additions to the BODA machine:
Boda

Additions to the Hot Drinks machine:
sdhjseg

Sodas that were added/changed:
soda

New Changeling Sting sprite animation (base .gif - not actual animation framerate):
changeling sting2

New Carnoth (ntcahors) Bottle:
asd

Misc Stuff:
-Sodawater and Tonic cans were brought to 30 from 50 to match other cans.
-Adds in previously unused Soda Bottles found in `drinks.dmi`.
-Moves Grenadine next to Lemon and Lime juice bottles in the code.
-Addition of Grape Juice for non-alcoholic characters that wish to be included.
-Addition of Turnip Juice reagent for those that wish to taste sadness.

### CODE REVIEW

**Comment by Greenjoe12345** [comment]
the radren-vend has an error on it's animation

**Comment by GhostActual** [comment]
> the radren-vend has an error on it's animation

Fixed Error

File: code/modules/reagents/reagents/food_drinks.dm [hunk 1]
```diff
@@ -1714,6 +1727,110 @@
...
+	name = "Strawberry Soda"
+	id = "strawsoda"
+	description = "Soda made using sweet berries."
+	taste_description = "oddly bland"
+	reagent_state = LIQUID
+	color = "#ffa3a3"
+	adj_drowsy = -3
+	adj_temp = -5
+
+	glass_name = "Orange Soda"
```
Comment (Greenjoe12345) [inline code/modules/reagents/reagents/food_drinks.dm:1771]: this should be the strawberry soda's stuff

Comment (GhostActual) [reply code/modules/reagents/reagents/food_drinks.dm:1771]: Fixed Error

**Comment by Atermonera** [comment]
Sorry for the delays here, ater has been busy with work.
CI finds ` [2021-07-24T17:15:05] Runtime in unsorted.dm,1331: The chemical dispenser cartridge attempted to add a reagent called 'wine' which doesn't exist. ()`

**Comment by GhostActual** [comment]
Redo

**Pull request closed without merging**